### PR TITLE
OCPBUGS-34644: Adding IP address to example

### PR DESCRIPTION
**Pull request by rohennes**
OCPBUGS-34644: If the IP address for the interface isn't already defined, it will cause issues with the procedure. 

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-34644

Link to docs preview:
https://76690--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-configure-return-traffic.html#nw-metallb-configure-return-traffic-proc_metallb-configure-return-traffic

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-metallb-configure-return-traffic-proc.adoc [hunk 1]
```diff
@@ -40,37 +42,47 @@ spec:
...
-          route-table-id: 2 <6>
-    routes: <7>
+    - name: ens4vrf <3>
+      type: vrf <4>
+      state: up
+      vrf:
+        port:
+        - ens4 <5>
+        route-table-id: 2 <6>
+    - name: ens4 <7>
```
Comment (rohennes) [inline modules/nw-metallb-configure-return-traffic-proc.adoc:52]: @Timdawson264 - I updated the indentation for the `route-table-id: 2` field so hopefully this is good now? I tried creating this in a metal clusterbot instance and it applied successfully.

**Comment by rohennes** [comment]
Fixes #76403

**Comment by rohennes** [comment]
@asood-rh - could you please QE review this [bug](https://issues.redhat.com/browse/OCPBUGS-34644) reported by Tim?

File: modules/nw-metallb-configure-return-traffic-proc.adoc [hunk 2]
```diff
@@ -40,37 +42,47 @@ spec:
...
 ----
 <1> The name of the policy.
 <2> This example applies the policy to all nodes with the label `vrf:true`.
 <3> The name of the interface.
 <4> The type of interface. This example creates a VRF instance.
 <5> The node interface that the VRF attaches to.
 <6> The name of the route table ID for the VRF.
-<7> Defines the configuration for network routes. The `next-hop-address` field defines the IP address of the next hop for the route. The `next-hop-interface` field defines the outgoing interface for the route. In this example, the VRF routing table is `2`, which references the ID that you define in the `EgressService` CR.
-<8> Defines additional route rules. The `ip-to` fields must match the `Cluster Network` CIDR and `Service Network` CIDR. You can view the values for these CIDR address specifications by running the following command: `oc describe network.config/cluster`.
+<7> In this example, this stanza defines the IPv4 address of the interface associated with the VRF.
```
Comment (asood-rh) [inline modules/nw-metallb-configure-return-traffic-proc.adoc:83]: Is it the IP address configured on the node?

Comment (rohennes) [reply modules/nw-metallb-configure-return-traffic-proc.adoc:83]: @Timdawson264 / @fedepaol - could help help with Arti's question please?

Comment (Timdawson264) [reply modules/nw-metallb-configure-return-traffic-proc.adoc:83]: yea <7> should get added by the NNCP to the node interface,we need the IP to use for l3 routing. i think its worth noting that route-table: 254 is the main table. not everyone knows that 254 is a special number.

Comment (rohennes) [reply modules/nw-metallb-configure-return-traffic-proc.adoc:83]: Updated with callout about routing table 254

File: modules/nw-metallb-configure-return-traffic-proc.adoc [hunk 3]
```diff
@@ -40,37 +42,48 @@ spec:
...
-<8> Defines additional route rules. The `ip-to` fields must match the `Cluster Network` CIDR and `Service Network` CIDR. You can view the values for these CIDR address specifications by running the following command: `oc describe network.config/cluster`.
+<7> In this example, this stanza defines the IPv4 address of the interface associated with the VRF. 
+<8> Defines the configuration for network routes. The `next-hop-address` field defines the IP address of the next hop for the route. The `next-hop-interface` field defines the outgoing interface for the route. In this example, the VRF routing table is `2`, which references the ID that you define in the `EgressService` CR.
+<9> Defines additional route rules. The `ip-to` fields must match the `Cluster Network` CIDR and `Service Network` CIDR. You can view the values for these CIDR address specifications by running the following command: `oc describe network.config/cluster`.
+<10> The route table with the ID `254` is the main routing table that the Linux kernel uses when calculating routes.
```
Comment (bburt-rh) [inline modules/nw-metallb-configure-return-traffic-proc.adoc:86]: [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), item 10 needs to be reworded to be grammatically parallel (i.e. a sentence fragment) with the other items in the callout list.

File: modules/nw-metallb-configure-return-traffic-proc.adoc [hunk 4]
```diff
@@ -40,37 +42,48 @@ spec:
...
 ----
 <1> The name of the policy.
 <2> This example applies the policy to all nodes with the label `vrf:true`.
 <3> The name of the interface.
 <4> The type of interface. This example creates a VRF instance.
 <5> The node interface that the VRF attaches to.
 <6> The name of the route table ID for the VRF.
-<7> Defines the configuration for network routes. The `next-hop-address` field defines the IP address of the next hop for the route. The `next-hop-interface` field defines the outgoing interface for the route. In this example, the VRF routing table is `2`, which references the ID that you define in the `EgressService` CR.
-<8> Defines additional route rules. The `ip-to` fields must match the `Cluster Network` CIDR and `Service Network` CIDR. You can view the values for these CIDR address specifications by running the following command: `oc describe network.config/cluster`.
+<7> In this example, this stanza defines the IPv4 address of the interface associated with the VRF.
```
Comment (bburt-rh) [inline modules/nw-metallb-configure-return-traffic-proc.adoc:83]: [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), item 7 needs to be reworded to be grammatically parallel (i.e. a sentence fragment) with the other items in the callout list.

**Review by bburt-rh — COMMENTED** [review]
A couple of nitpicks about making the callout list items all grammatically parallel, but otherwise LGTM

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: modules/nw-metallb-configure-return-traffic-proc.adoc [hunk 5]
```diff
@@ -40,37 +42,48 @@ spec:
...
-          next-hop-address: 192.168.130.1
-          next-hop-interface: ens4
-          table-id: 2
-    route-rules: <8>
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 192.168.130.1
+        next-hop-interface: ens4
+        table-id: 2
+    route-rules: <9>
```
Comment (Timdawson264) [inline modules/nw-metallb-configure-return-traffic-proc.adoc:68]: can you add another block to route-rules
this is the ovn host-to-service masquerade subnet its sometimes used for external IP -> service natting and needs a rule for vrfs to ingress traffic 
```
- ip-to: 169.254.169.0/29
          priority: 998
          route-table: 254
```

**Pull request merged**